recipes/psychopy/meta.yaml

```diff
@@ -0,0 +1,108 @@
+{% set name = "psychopy" %}
+{% set pypi_name = "PsychoPy" %}
+{% set version = "3.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.zip
+  sha256: 5df7c0ff494368fcc4ade820d5b83135f82a44740868ed199d1023e181c30f95
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install --no-deps . -vv"
+  entry_points:
+    - psychopy = psychopy.app.psychopyApp:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - {{ cdt('mesa-libgl') }}        # [linux]
```

Comment:
Is this a pure Python package? If so, you don't need CDTs and instead only need OpenGL to be installed on the host. See the `yum_requirements.txt` [here](http://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=mesa#core-dependency-tree-packages-cdt).

Reply:
Thank you, I've done that now.

Reply:
You're missing some lines from `yum_requirements.txt ` and this line isn't needed:
```suggestion
```

Reply:
Whoops. I missed that. Updating now!